nexus/src/external_api/http_entrypoints.rs

```diff
@@ -2286,19 +2286,21 @@ async fn networking_address_lot_create(
 /// Delete an address lot.
 #[endpoint {
     method = DELETE,
-    path = "/v1/system/networking/address-lot",
+    path = "/v1/system/networking/address-lot/{address_lot}",
     tags = ["external-networking"],
 }]
 async fn networking_address_lot_delete(
     rqctx: RequestContext<Arc<ServerContext>>,
-    query_params: Query<params::AddressLotSelector>,
+    path_params: Path<params::AddressLotPath>,
 ) -> Result<HttpResponseDeleted, HttpError> {
```

Comment:
The API only uses selectors to help narrow down resources that may have a parent container. For example, if you're looking up an instance you specify the instance in the path (`/v1/instances/{instance}`). If you pass a `Name` for `instance`, we don't have enough information to do a lookup (because instance names are scoped to projects) so you need to narrow it with a selector. Like `/v1/instances/{instance_name}?project={project}`. If `instance` is provided by an ID no selector is needed so the call is just `/v1/instances/{instance_id}`.